storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -1,51 +1,51 @@
 
-To create CSI-provisioned PVs that mount to AWS EBS storage assets, {product-title} installs the AWS EBS CSI Driver Operator and the AWS EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+To create CSI-provisioned PVs that mount to Amazon EBS storage assets, {product-title} installs the Amazon EBS CSI Driver Operator and the Amazon EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
 
-* The _AWS EBS CSI Driver Operator_ provides a StorageClass by default that you can use to create PVCs. You can disable this default storage class if desired (see xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#persistent-storage-csi-sc-manage[Managing the default storage class]). You also have the option to create the AWS EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using AWS Elastic Block Store]. 
+* The _Amazon EBS CSI Driver Operator_ provides a StorageClass by default that you can use to create PVCs. You can disable this default storage class if desired (see xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#persistent-storage-csi-sc-manage[Managing the default storage class]). You also have the option to create the Amazon EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using Amazon Elastic Block Store]. 
 
-* The _AWS EBS CSI driver_ enables you to create and mount AWS EBS PVs.
+* The _Amazon EBS CSI driver_ enables you to create and mount Amazon EBS PVs.
 
 ifndef::openshift-rosa,openshift-dedicated[]
 [NOTE]
 ====
-If you installed the AWS EBS CSI Operator and driver on an {product-title} 4.5 cluster, you must uninstall the 4.5 Operator and driver before you update to {product-title} {product-version}.
+If you installed the Amazon EBS CSI Operator and driver on an {product-title} 4.5 cluster, you must uninstall the 4.5 Operator and driver before you update to {product-title} {product-version}.
 ====
 endif::openshift-rosa,openshift-dedicated[]
 
 include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
-{product-title} defaults to using the CSI plugin to provision AWS EBS storage.
+{product-title} defaults to using the CSI plugin to provision Amazon EBS storage.
 ====
 
-For information about dynamically provisioning AWS EBS persistent volumes in {product-title}, see xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using AWS Elastic Block Store].
+For information about dynamically provisioning Amazon EBS persistent volumes in {product-title}, see xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using Amazon Elastic Block Store].
```

Comment:
I have updated on Line 35. Please let me know if this instance need to be revoked. Thank you!

Reply:
Revoked.